Marlin/src/config/examples/delta/Tevo Little Monster/Configuration.h

```diff
@@ -1669,6 +1669,12 @@
 //
 //#define ENCODER_STEPS_PER_MENU_ITEM 1
 
+//
+// Use this if your LCD uses buttons instead of a dial encoder
+// Changes locations of some menus for usability
+//#define ENCODER_TYPE_BUTTONS
```

Comment:
This can be figured out logically without adding yet another top-level option in `Configuration.h`.